README.md

```diff
@@ -6,3 +6,14 @@ Strong hack
 Note:
 
 MacOS is unsupported and will NOT be fixed since it is an Apple issue.
+
+---
+
+# FAQ
+Before opening an issue, please consider the following:  
+- Is OptiFine installed? (If yes, disable it. If no, install it)
```

Comment:
I don't think Optifine causes issues with TrollHack?

Reply:
Any logs or infos on it?

Reply:
nah didnt really think too much of it, it works now

Reply:
Then you should remove that line